icu4j/main/classes/core/src/com/ibm/icu/text/NumberFormatServiceShim.java

```diff
@@ -93,8 +93,11 @@ NumberFormat createInstance(ULocale desiredLocale, int choice) {
         // If we are creating a currency type formatter, then we may have to set the currency
         // explicitly, since the actualLoc may be different than the desiredLocale        
         if ( choice == NumberFormat.CURRENCYSTYLE ||
-             choice == NumberFormat.ISOCURRENCYSTYLE || 
-             choice == NumberFormat.PLURALCURRENCYSTYLE) {
+             choice == NumberFormat.ISOCURRENCYSTYLE ||
```

Comment:
LGTM, though I wonder if `NumberFormat.isCurrency()` might make sense for Java?

Reply:
@srl295 
Sorry I can't find any NumberFormat#isCurrency on neither ICU4J nor JDK.
Did you meant that we should refactor the logic out to a utility method?
I would agree if similar logic exist elsewhere, but this is only a local code change :)

Reply:
No, there's no such method. I meant it could be added as a private method on the enum.  But, it's probably not needed. Anyway I accept this.

Reply:
@srl295 Great! Thanks!